cognite_toolkit/_cdf.py

```diff
@@ -56,39 +65,28 @@
         traces_sample_rate=1.0,
     )
 
+default_typer_kws = dict(
+    pretty_exceptions_short=False, pretty_exceptions_show_locals=False, pretty_exceptions_enable=False
+)
 try:
-    app = typer.Typer(
-        pretty_exceptions_short=False, pretty_exceptions_show_locals=False, pretty_exceptions_enable=False
-    )
+    typer.Typer(**default_typer_kws)
 except AttributeError as e:
     # From Typer version 0.11 -> 0.12, breaks if you have an existing installation.
-    print(
-        "'cognite-toolkit' uses a dependency named 'typer'.",
-        "From 'typer' version 0.11 -> 0.12 there was a breaking change if you have an existing "
-        "installation of 'typer'.",
-        "Workaround is to uninstall 'typer-slim', and then, reinstall 'typer':",
-        "'pip uninstall typer-slim'",
-        "'pip install typer'",
-        sep="\n",
+    raise ToolkitError(
```

Comment:
Here we might have to do something with `sentry`. If we raise regular errors they will be picked up by `sentry` and reported. This type of error we do not want `sentry` to report on.

Reply:
Addressed by adding a `before_send=sentry_exception_filter` function to filter out our own exceptions